source/plain.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+const plain_inner = (result, array)=>{
```

Comment:
Форматирование: не стоит экономить пробелы и добавлять разделение между символами
```suggestion
const plain_inner = (result, array) => {
```